### PR TITLE
Python API For compare_functions

### PR DESCRIPTION
**Pull request by GlebKazantaev**
This PR includes:
* Python API for `compare_functions`
* Fixed graph traversal to avoid excess nodes comparison
* Fixed Constant values comparison to use vector of values instead of string with values that previously took an eternity.
* Added python API tests

### CODE REVIEW

File: inference-engine/ie_bridges/python/src/openvino/offline_transformations/CMakeLists.txt [hunk 1]
```diff
@@ -25,9 +25,9 @@ if(COMMAND ie_add_vs_version_file)
 endif()
 
 if(TARGET offline_transformations)
-    list(APPEND InferenceEngine_LIBRARIES offline_transformations)
+    list(APPEND InferenceEngine_LIBRARIES offline_transformations commonTestUtils)
 else()
-    list(APPEND InferenceEngine_LIBRARIES IE::offline_transformations)
+    list(APPEND InferenceEngine_LIBRARIES IE::offline_transformations IE::commonTestUtils)
```
Comment (ilya-lavrenov) [inline inference-engine/ie_bridges/python/src/openvino/offline_transformations/CMakeLists.txt:30]: I supposed that python offline API is release component, while `commonTestUtils` and other test utils - does not. 

Do we need it only for usage in tests?

Comment (GlebKazantaev) [reply inference-engine/ie_bridges/python/src/openvino/offline_transformations/CMakeLists.txt]: Moved to a separate python module.

**Review by ilya-lavrenov — APPROVED** [review]

File: inference-engine/ie_bridges/python/CMakeLists.txt [hunk 2]
```diff
@@ -55,6 +55,10 @@ set (PYTHON_BRIDGE_SRC_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
 add_subdirectory (src/openvino/inference_engine)
 add_subdirectory (src/openvino/offline_transformations)
 
+if (TARGET commonTestUtils OR TARGET IE::commonTestUtils)
```
Comment (ilyachur) [inline inference-engine/ie_bridges/python/CMakeLists.txt:58]: Can we change it to: `if (ENABLE_TESTS)`?

**Review by akuporos — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

File: inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp [hunk 3]
```diff
@@ -149,11 +152,11 @@ std::pair<bool, std::string> compare_functions(
                 auto const2 = ngraph::as_type_ptr<Constant>(node2->get_input_node_shared_ptr(i));
 
                 const auto equal = [](const Constant &c1, const Constant &c2) {
-                    const auto equal_float_str = [](const std::string &s1, const std::string s2) {
-                        return std::abs(std::stof(s1) - std::stof(s2)) < 0.001;
+                    const auto equal_float_str = [](const double &s1, const double & s2) {
```
Comment (pelszkow) [inline inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp:155]: `std::string`s are not compared now. Name is misleading.

Comment (GlebKazantaev) [reply inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp]: done.

**Pull request merged**